### PR TITLE
Docs/gail edits

### PR DESCRIPTION
**Pull request by jmlarson1**

### CODE REVIEW

File: docs/tutorials/local_sine_tutorial.rst [hunk 1]
```diff
@@ -128,7 +128,7 @@ Simulator function
 Next, we'll write our :ref:`sim_f<api_sim_f>` or simulator function. Simulator
 functions perform calculations based on values from the generator function.
 The only new parameter here is :ref:`sim_specs<datastruct-sim-specs>`, which serves
-a similar purpose to ``gen_specs``.
+a purpose similar that for to ``gen_specs``.
```
Comment (shuds13) [inline docs/tutorials/local_sine_tutorial.rst]: similar to that of?

File: docs/tutorials/local_sine_tutorial.rst [hunk 2]
```diff
@@ -233,9 +233,9 @@ That's it! Now that these files are complete, we can run our simulation.
 
   $ python3 calling_script.py
 
-If everything ran perfectly and you included the above print-statements, You
+If everything ran perfectly and you included the above print statements, you
 should get something similar to the following output for ``H``. The columns might
-be rearranged.
+be rearranged as follows.
```
Comment (shuds13) [inline docs/tutorials/local_sine_tutorial.rst]: (the columns may be arranged differently) - ie. this is saying that the order of fields shown below is not necessarily going to be the same.

Comment (jmlarson1) [reply docs/tutorials/local_sine_tutorial.rst]: Doesn't the "might be rearranged as follows" imply this?

File: docs/tutorials/local_sine_tutorial.rst (hunk 1 above)
Comment (jmlarson1) [reply docs/tutorials/local_sine_tutorial.rst]: Does the new wording work: "serves a purpose similar to the ``gen_specs`` dictionary." ?

File: docs/tutorials/local_sine_tutorial.rst (hunk 2 above)
Comment (jlnav) [reply docs/tutorials/local_sine_tutorial.rst]: The order of the columns don't appear to be predictable. The provided H sample is just one of many possible combinations. The order "might" be arranged "as follows", but it's a small chance.

Maybe "The output columns may be arranged differently." ? I'm trying to communicate the impression that as long as all rows and columns are included everything worked fine, and columns being rearranged into whichever-order is normal.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -27,55 +29,55 @@ Other changes:
...
 
-* Code restructured to meet xSDK package policies for interoperable ECP software (version 0.5.0). #208
-* Removed use of MPI.COMM_WORLD. Uses a duplicate of COMM_WORLD if no communicator passed (any process not in communicator returns with an exit code of 3). #108
+* Code has been restructured to meet xSDK package policies for interoperable ECP software (version 0.5.0). #208
+* The use of MPI.COMM_WORLD has been removed. Uses a duplicate of COMM_WORLD if no communicator passed (any process not in communicator returns with an exit code of 3). #108
 * All output from libEnsemble goes via logger. MANAGER_WARNING level added. This level and above are echoed to stderr by default. API option to change echo level.
-* Simulation directories are only created as required and are suffixed by _worker. #146
-* New user function libE.check_inputs() can be used to check valid configuration of inputs. Can be called in serial or under MPI (See libE API). #65
-* Installation option added to install dependencies used in tests ``pip install libensemble[extras]``
+* Simulation directories are created only as required and are suffixed by _worker. #146
```
Comment (jlnav) [inline CHANGELOG.rst]: Can reduce vague-ness here?

```suggestion
* Simulation directories are only created during sim_f calls and are suffixed by _worker. #146
```

File: docs/FAQ.rst [hunk 4]
```diff
@@ -96,12 +97,12 @@ nodes, while launching all sub-jobs onto other nodes.
 
 **What does "_pickle.UnpicklingError: invalid load key, '\x00'." indicate?**
 
-This has been observed with the OFA fabric, and usually indicates MPI messages
+This has been observed with the OFA fabric and usually indicates MPI messages
```
Comment (jlnav) [inline docs/FAQ.rst]: Move error condition to earlier in line.

```suggestion
This has been observed with the OFA fabric when using mpi4py and usually indicates MPI messages
```

File: docs/FAQ.rst [hunk 5]
```diff
@@ -96,12 +97,12 @@ nodes, while launching all sub-jobs onto other nodes.
...
 aren't being received correctly when using mpi4py. The solution
```
Comment (jlnav) [inline docs/FAQ.rst]:
```suggestion
 aren't being received correctly. The solution
```

File: docs/platforms/bebop.rst [hunk 6]
```diff
@@ -2,7 +2,8 @@
 Bebop
 =====
 
-Bebop_ is a Cray CS400 cluster available within LCRC at Argonne National
+Bebop_ is a Cray CS400 cluster available in the Laboratory Computing Resouces
```
Comment (jlnav) [inline docs/platforms/bebop.rst]:
```suggestion
Bebop_ is a Cray CS400 cluster with Intel Broadwell and Knights Landing compute nodes available in the Laboratory Computing Resources
```

File: docs/platforms/bebop.rst [hunk 7]
```diff
@@ -2,7 +2,8 @@
...
+Center (LCRC) at Argonne National
 Laboratory, featuring both Intel Broadwell and Knights Landing compute nodes.
```
Comment (jlnav) [inline docs/platforms/bebop.rst]:
```suggestion
Laboratory.
```

File: docs/platforms/bebop.rst [hunk 8]
```diff
@@ -57,30 +58,30 @@ With your nodes allocated, queue your job to start with four MPI ranks::
...
 
 .. note::
     When performing a :doc:`distributed<platforms_index>` MPI libEnsemble run
     and not oversubscribing, specify one more MPI process than the number of
-    allocated nodes. The Manager and first worker run together on a node.
+    allocated nodes. The manager and first worker run together on a node.
 
 If you would like to interact directly with the compute nodes via a shell,
-the following showcases starting a bash session on a Knights Landing node
+the following shows starting a bash session on a Knights Landing node
```
Comment (jlnav) [inline docs/platforms/bebop.rst]:
```suggestion
the following starts a bash session on a Knights Landing node
```

File: docs/tutorials/local_sine_tutorial.rst (hunk 1 above)
Comment (jlnav) [reply docs/tutorials/local_sine_tutorial.rst]: That wording works for me.

**Pull request merged**